make/framework.mk

```diff
@@ -95,14 +95,14 @@ framework:: lib $(FRAMEWORK_DIR) resources
 # Create an xcframework from all appletv, iphone, maccatalyst, macosx, simulator and watchos libs.
 $(FRAMEWORK_DIR): lib $(FRAMEWORK_HEADER) | $(DIST_FRAMEWORK_DIR)
 	@echo building $(FRAMEWORK_NAME) framework
-	@rm -rf ${FRAMEWORK_DIR}
-	@mkdir -p $(FRAMEWORK_DIR)/Headers
```

Comment:
The `$@` variable resolves to the name of the target being built and is thus synonymous with `$(FRAMEWORK_DIR)` here. I'm opting to use the `$@` variable rather than `$(FRAMEWORK_DIR)` to bring consistency across this file. Right now we're arbitrarily using `$@` in some places and not in others.

Reply:
SGTM